### DISCUSSION TITLE
Auto-center components on canvas

### DISCUSSION THREAD

**Discussion by josejbracho** [GrapesJS/grapesjs#5465] [Q&A]
Hi,

In dragMode: 'absolute', I was wondering if there is a way to drop a component inside the canvas and have it 'auto-center'.

Is there a way to manipulate the positioning of an element after it's dropped inside the canvas?

Thanks,

Jose

**Comment by badiuciprian** [comment]
One way to do it is by css. Since they are absolute you could use .myComponent {left: 50%, transform: translateX: -50%}
